### ISSUE TITLE
Question: call help with different formatter

### ISSUE THREAD

**Issue by emKaroly** [CLIUtils/CLI11#419]
I created a custom formatter and i want to print help with that formatter based on which flag i set.
--help-all would print with default formatter
--help-all-custom print with custom formatter

is that possible? 

Thank you.

**Comment by henryiii** [comment]
Not exactly, but you can set up your custom formatter to call the original formatter if help-all is not passed. I can try to make a little example later if that's helpful.

I would probably inherit from Formatter, then call Formatter::x if AppFormatMode is not help_all. Or you can make it a member of your formatter and call it manually.
